### DISCUSSION TITLE
Penzias and Wilson doubted CMB as Big Bang relic

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#135] [Astrophysics and Physics]
> “I think both Arnold and I wanted to leave open the idea that there was some other source of this noise,” he added. “But, of course, that didn’t work out.”

https://www.nytimes.com/2023/09/04/science/astronomy-holmdel-antenna-microwaves.html

**Comment by RedshiftDrift** [comment]
I hope they keep the horn as a historic monument, a reminder that our science can be thrown off course by a misinterpreted observation.

**Comment by HanDeBruijn** [comment]
In what follows, three references are made to posts in discussion 127 named [_Can the CMB temperature be less than 2.7 K ?_](https://github.com/orgs/a-cosmology-group/discussions/127)
Important for our theory is the following function, derived in the [first reference](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6812985).

$$
f(x) = \frac{2-e^{-x}(x^2+2x+2)}{x} \quad \mbox{where} \quad x = \frac{c}{H}.R
$$

$x$ is a dimensionless variable, with $c=$ lightspeed, $H=$ Hubble parameter, $R=$ distance.
![sahil2](https://github.com/a-cosmology-group/acg/assets/135433222/aa073f84-1808-436b-bc0c-574bb4d93fbf)
All values of the function must be below a certain value. For e.g. the &Lambda;CDM model we have found that $f(x) \lt 1/3$ . For other values of $x$ that part of the universe is a Black Hole, meaning that no light from there can enter our cosmic bubble. We cannot observe anything where $f(x) \gt 1/3$, according to &Lambda;CDM. However, alternative cosmic models can be devised. The most extreme one is defined by calculating the maximum of the function $f(x)$. Done numerically:
```
f(x) := (2-exp(-x)*(x^2+2*x+2))/x;
diff(f(x),x); s := fsolve(%=0,{x});
assign(s);
w := (2-exp(-x)*(x^2+2*x+2))/x;

                        s := {x = 3.383634283}

                          w := 0.3883945571
```
This results in a condition $f(x) \lt 0.3883945571$, a bit less restrictive than &Lambda;CDM, with far reaching consequences though.
In a [second reference](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6826896) we found for the total mass of our **infinite SEU** (Static Euclidean Universe) with $G=$ gravitational constant:

$$
M_U = \frac{c^3}{w.G.H} \quad \mbox{where} \quad w = 0.3883945571
$$

There is also the mass of our Observable universe, which is - easy to derive - related to the mass of the whole universe as

$$
M_O = \frac{W.w}{2} M_U = \frac{W}{2}\frac{c^3}{G.H} \quad \mbox{where} \quad W = 3.383634283
$$

And the radius of our Observable universe is

$$
R_O = W\frac{c}{H}
$$

In a [third reference](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6891995) we found for the gravitational self energy density:

$$
u_G = - GM^2/(8\pi r^4)
$$

Integrating over our Observable universe:

$$
U_G = \int_{R_O}^\infty \left[ - \frac{G M_O^2}{8\pi r^4} 4\pi r^2 \right] dr = - \frac{G M_O^2}{2}\int_{R_O}^\infty \frac{dr}{r^2} = - \frac{G M_O^2}{2R_O}
$$

The latter outcome is also known from the booklet [Relativity Reexamined](http://www.alternatievewiskunde.nl/einstein/relativity-reexamined_compress.pdf) by [L&eacute;on Brillouin](https://en.wikipedia.org/wiki/L%C3%A9on_Brillouin): formula (7.13) times $c^2$.
But now it goes wrong because, due to $E=mc^2$, _energy has a mass_. Therefore we must integrate the energy density according to the Variable Mass Theory; we cannot assume that it's a constant. The total gravitational self energy of the observable universe herewith becomes

$$
U_G = - \frac{G M_O^2}{2} \int_{R_O}^\infty \frac{e^{-H/c.r}}{r^2}dr = - \frac{G M_O^2}{2R_O}.W.\mbox{VMT}
$$

Computing the integral by hand is a bit difficult, but we let a Computer Algebra System (MAPLE) do the work.

$$
\mbox{VMT} = \int_W^\infty \frac{e^{-x}}{x^2}dx = \frac{e^{-W}}{W} - \mbox{Ei}(W) \quad \mbox{with} \quad \mbox{Ei}(x) = \int_1^\infty \frac{e^{-xt}}{t}dt
$$

With a slight modification it is assumed again that

### Uniform CMB energy + Gravitational self energy = 0

Where

$$
U_{CMB} = \frac{4}{3}\pi R_O^3.\frac{4\sigma}{c}\Theta^4
$$

And so

$$
\frac{4\sigma}{c}\Theta^4 = \frac{G M_O^2}{2R_O.4/3.\pi R_O^3} W.\mbox{VMT} = \frac{3.\mbox{VMT}}{32\pi.W} \frac{(H.c)^2}{G}
$$

![to_bud2](https://github.com/a-cosmology-group/acg/assets/135433222/48bcadbf-896c-4fc9-b513-700f78872526)
```
W := 3.383634283;
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H := 73.4; # Riess
H := H*1000/Mpc;
c := 299792458;
VMT := int(exp(-x)/x^2,x=W..infinity);
u_G := (3*VMT)/(32*Pi*W)*(H*c)^2/G;
sigma := 5.670374419*10^(-8);
Theta := evalf((c/(4*sigma)*u_G)^(1/4));

                         Theta := 3.638225492 # Kelvin
```
**No** expanding space-time, **no** Big Bang, **no** Co(s)mic Inflation, **no** Primordial Relic Radiation, **no** fuzz parameters,  **no** tweaking, no nothing of the kind. And yet a reasonable estimate of the Cosmic Microwave Background &Theta;emperature.

**Reply by sahil5d** [reply]
It's worth noting that the CMB power/area is similar to the power/area of the Milky Way...

CMB
~ 3 microWatts / m^2
https://wolframalpha.com/input?i=stefan+boltzmann+constant+*+%282.7+K%29%5E4

Milky Way
~ 9 microWatts / m^2
https://wolframalpha.com/input?i=milky+way+luminosity+%2F+%284pi+*+milky+way+radius%5E2%29

... as well as neighboring galaxies...

Andromeda
~ 2 microWatts / m^2
https://wolframalpha.com/input?i=+%283.3+*+10%5E10+solar+luminosity%29+%2F+%284pi+*+%28radius+of+andromeda+galaxy%29%5E2%29

Pinwheel Galaxy
~ 1 microWatt / m^2
https://wolframalpha.com/input?i=%2830+billion+*+solar+luminosity%29+%2F+%284pi+*+%2885000+light+year%29%5E2%29

Whirlpool Galaxy
~ 2 microWatts / m^2
https://wolframalpha.com/input?i=%2810+*+10%5E9+*+luminosity+of+sun%29+%2F+%284pi+*+%2838000+lightyear%29%5E2%29

Sunflower Galaxy
~ 18 microWatts / m^2
https://wolframalpha.com/input?i=%281.9+*+milky+way+luminosity%29+%2F+%284pi+*+%2849000+lightyear%29%5E2%29

Messier 60
~ 6 microWatts / m^2
https://wolframalpha.com/input?i=%2860+*+10%5E9+solar+luminosity%29+%2F+%284pi+*+%2860000+light+year%29%5E2%29

My guess is that the CMB is a supercluster-sized (~ .1 Gly radius) phenomenon.
